Source/ZXing.Net.Mobile.Android/ZXingSurfaceView.cs

```diff
@@ -166,8 +166,29 @@ void OnSurfaceChanged ()
 
             var parameters = camera.GetParameters ();
             parameters.PreviewFormat = ImageFormatType.Nv21;
-            parameters.SetPreviewFpsRange(15, 30);
```

Comment:
This line in master branch now causes exception because this range is not listed in supported fps ranges (at least for my device :smile: ).